UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Program.cs

```diff
@@ -78,6 +92,29 @@ private static HostBuilder BuildHostConfiguration()
                  {
                      builder.AddApplicationInsightsWebJobs(o => o.InstrumentationKey = instrumentationKey);
                  }
+                 var eventhubConfig = ConfigurationBuilder.GetSection("EventHubLoggingConfiguration");
+
+                 if (!string.IsNullOrWhiteSpace(eventhubConfig["ConnectionString"]))
+                 {
+                     builder.AddEventHub(config =>
+                     {
+                     config.Environment = eventhubConfig["Environment"];
+                     config.DefaultMinimumLogLevel =
+                         (LogLevel)Enum.Parse(typeof(LogLevel), eventhubConfig["MinimumLoggingLevel"], true);
+                     config.MinimumLogLevels["UKHO"] =
+                         (LogLevel)Enum.Parse(typeof(LogLevel), eventhubConfig["UkhoMinimumLoggingLevel"], true);
+                     config.EventHubConnectionString = eventhubConfig["ConnectionString"];
+                     config.EventHubEntityPath = eventhubConfig["EntityPath"];
+                     config.System = eventhubConfig["System"];
+                     config.Service = eventhubConfig["Service"];
+                     config.NodeName = eventhubConfig["NodeName"];
+                     config.AdditionalValuesProvider = additionalValues =>
+                         {
+                            additionalValues["_AssemblyVersion"] = Assembly.GetExecutingAssembly().GetCustomAttributes<AssemblyFileVersionAttribute>().Single().Version;
```

Comment:
This version string is used several times in the `BuildHostConfiguration` method. Could it be pulled out as a variable?

Reply:
As per suggestion, I have declared version string as a variable.